src/UI/Counter/styles.ts

```diff
@@ -33,8 +33,9 @@ export const styles = StyleSheet.create({
   },
   count: {
     fontSize: 20,
+    fontFamily: 'Montserrat',
```

Comment:
it's a good practice to have the same approach for fonts as you have for colours - Font.primary, Font.secondary